shared/src/main/scala/mlscript/Typer.scala

```diff
@@ -1068,15 +1068,29 @@ class Typer(var dbg: Boolean, var verbose: Bool, var explainErrors: Bool, var ne
                 rcdSel(realPrefix, fieldName)
               }
           }
+        def fallback =
```

Comment:
Document that this is a temporary solution to the lack of proper static `unapply` method desugaring from class definitions.

Reply:
Added comments for the temporary solution.